admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/TemplateServiceImpl.java

```diff
@@ -78,6 +83,15 @@
 
 	@Value("#{'${mosip.mandatory-languages}'.concat('${mosip.optional-languages}')}")
 	private String supportedLang;
+	
+	@Value("${mosip.kernel.masterdata.template_idauthentication_event:masterdata/idauthentication_templates}")
```

Comment:
can we have topic dynamically constructed as: masterdata/templates/{module-name}